stories/modules-tagmanager-settings.stories.js

```diff
@@ -389,104 +325,6 @@ storiesOf( 'Tag Manager Module/Settings', module )
 		{
 			decorators: [ withRegistry ],
 		}
-	)
-	.add(
```

Comment:
Likewise, let's keep one of these _Edit..._ stories and remove the access related stuff from it.